modules/installation-azure-config-yaml.adoc

```diff
@@ -108,25 +114,34 @@ If you disable simultaneous multithreading, ensure that your capacity planning a
 //storage type as `io1` and set `iops` to `2000`.
 <6> Specify a list of zones to deploy your machines to. For high availability, specify at least two zones.
 <7> Specify the name of the resource group that contains the DNS zone for your base domain.
-ifdef::vnet[]
+ifdef::vnet,private[]
 <8> If you use an existing VNet, specify the name of the resource group that contains it.
 <9> If you use an existing VNet, specify its name.
 <10> If you use an existing VNet, specify the name of the subnet to host the control plane machines.
 <11> If you use an existing VNet, specify the name of the subnet to host the compute machines.
+endif::vnet,private[]
+ifdef::private[]
```

Comment:
Yikes, I can see why you were interested in that auto callout numbering!!

Reply:
Yeah, these config files are getting scary :dizzy_face: